src/com_tjnotifications/admin/language/en-GB.com_tjnotifications.ini

```diff
@@ -293,3 +293,9 @@ COM_TJNOTIFICATIONS_SUBSCRIPTIONS_PHONE_NUMBER_JOOMSOCIAL="JoomSocial"
 COM_TJNOTIFICATIONS_SUBSCRIPTIONS_SELECT_PHONE_NUMBER_FIELD="Select phone number field"
 COM_TJNOTIFICATIONS_SUBSCRIPTIONS_SELECT_PHONE_NUMBER_FIELD_DESC="Select Phone number field to get phone number"
 COM_TJNOTIFICATIONS_SUBSCRIPTIONS_SELECT_FIELD="-Select-"
+COM_TJNOTIFICATIONS_SETTINGS_URL_SHORTING="Enable URL shorting?"
+COM_TJNOTIFICATIONS_SETTINGS_URL_SHORTING_DESC="If set yes, URL from notification will short"
+COM_TJNOTIFICATIONS_SETTINGS_CHOOSE_URL_SHORTENING_PLUGIN="Choose URL shortening provider"
+COM_TJNOTIFICATIONS_SETTINGS_CHOOSE_URL_SHORTENING_PLUGIN_DESC="Choose URL shortening provider"
+COM_TJNOTIFICATIONS_SETTINGS_CHOOSE_BACKEND_SHORTENING="Choose backends for shortening URL"
+COM_TJNOTIFICATIONS_SETTINGS_CHOOSE_BACKEND_SHORTENING_DESC="Choose backends (like email,sms,whatsapp,push etc.) for shortening URL"
```

Comment:
add space after each comma

Reply:
What is backends?

Reply:
`(like email,sms,whatsapp,push etc.)` => `(like Email, SMS, WhatsApp, Push etc.)`